### ISSUE TITLE
W10 docker required?

### ISSUE THREAD

**Issue by LaravelFreelancerNL** [tpetry/stackbricks-community#23]
Docker seems to be required.

On the DB overview screen I get the error message: 'Docker service is unreachable!'

Any DB setup has an 'Unknown' status.

Deleting a DB setup gives the following docker related error:

```
Unhandled Promise Rejection
Error: Unknown Docker Error: connect ENOENT //./pipe/docker_engine
    at I3 ({path-placeholder}/StackBricks/resources/app/packages/main/dist/index.cjs:30:60490)
    at Er ({path-placeholder}/StackBricks/resources/app/packages/main/dist/index.cjs:30:60692)
    at du ({path-placeholder}/StackBricks/resources/app/packages/main/dist/index.cjs:30:60986)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async $a.delete ({path-placeholder}/StackBricks/resources/app/packages/main/dist/index.cjs:44:6249)
```

I'm running WSL2 and have a MySql 8 installed directly within it. To which I can connect from other W10 apps like HeidiSql.

**Comment by dvdheiden** [comment]
Yes, you need to have docker running if you want to use StackBricks. There's already an open issue to make that clearer upfront. See: https://github.com/tpetry/stackbricks-community/issues/14

**Comment by tpetry** [comment]
Yes, Docker is required and needs to be stated on the web page. But you shouldn‘t be able to do anything when docker is not running, I need to disable some actions in this use case.

**Comment by tpetry** [comment]
The bug that operations requiring docker can be done when docker is not running has been fixed with `0.5.1`. That the website should more clearly tell the docker requirement will be further tracked in #14.
